J/JSBSim/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+using BinaryBuilder
+
+name = "JSBSim"
+version = v"1.1.6"
+julia_version = v"1.5.3"
+
+# Collection of sources required to build JSBSim
+sources = [
+    GitSource("https://github.com/bcoconni/jsbsim.git",
+              "d699c3bcd90bf9ce2a64c1fe3091fcf39714ccfd"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/jsbsim
+mkdir build && cd build
+FLAGS=()
+if [[ "${target}" == *-mingw* ]]; then
+    FLAGS+=(-DCMAKE_CXX_FLAGS_RELEASE="-D_POSIX_C_SOURCE")
+fi
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_FIND_ROOT_PATH=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_DOCS=OFF \
+    -DBUILD_PYTHON_MODULE=OFF \
+    -DBUILD_JULIA_PACKAGE=ON \
+    -DJulia_PREFIX=${prefix} \
+    "${FLAGS[@]}" \
+    ..
+cmake --build . --target JSBSimJL -- -j${nproc}
+install_license $WORKSPACE/srcdir/jsbsim/COPYING
+cp julia/*JSBSimJL*.$dlext $libdir/.
+cp ../julia/JSBSim.jl $prefix/.
+"""
+
+ARGS
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+include("../../L/libjulia/common.jl")
+platforms = libjulia_platforms(julia_version)
+platforms = expand_cxxstring_abis(supported_platforms())
```

Comment:
That's wrong, you are discarding the result of `libjulia_platforms`.
```suggestion
platforms = expand_cxxstring_abis(platforms)
```